src/test/java/locators_constants/Constants.java

```diff
@@ -1,4 +1,4 @@
-package constants;
+package locators_constants;
```

Comment:
this class contains Test data, so it's not "locators_constants"

Reply:
done